src/http.rs

```diff
@@ -52,6 +52,7 @@ impl HttpError {
 }
 
 pub type HttpClientFuture = <HttpClient as Service<http::Request<Body>>>::Future;
+pub type BodyBox = http_body::combinators::BoxBody<hyper::body::Bytes, hyper::Error>;
```

Comment:
Since this type is now exported and used elsewhere, we should consider a more descriptive name, like `HttpBody`. That is, however, purely cosmetic at this point.

Reply:
Hmm there seems to be issues that arise when I change BodyBox to HttpBody namely:
HttpBody must be defined only once in the type namespace of this module. 

Assuming this means that hyper::body::HttpBody and the type alias for http::HttpBody can't co-exist in the same file.

Do you have any other suggestions for naming?

Reply:
Right, `HttpBody` is awkward as `hyper` exports a type of the same name. A fuller alternative name would be `HttpClientBody`.